components/eam/src/dynamics/se/dyn_comp.F90

```diff
@@ -112,6 +112,12 @@ subroutine dyn_init1(fh, NLFileName, dyn_in, dyn_out)
     integer :: npes_se
     integer :: npes_se_stride
 
+#ifdef KOKKOS_TARGET
+#ifdef _OPENMP
```

Comment:
Why did you choose _OPENMP rather than HORIZ_OPENMP and COLUMN_OPENMP?

Reply:
I decided that _OPENMP is more general condition, since $smp in configure triggers HORIZ_OPENMP, but I will change it to condition based on horizontal and column omp.